src/Features/VR.cpp

```diff
@@ -0,0 +1,60 @@
+﻿#include "VR.h"
+
+NLOHMANN_DEFINE_TYPE_NON_INTRUSIVE_WITH_DEFAULT(
+	VR::Settings,
+	EnableDepthBufferCulling,
+	MinOccludeeBoxExtent)
+
+void VR::DrawSettings()
+{
+	if (ImGui::Checkbox("Enable Depth Buffer Culling", &settings.EnableDepthBufferCulling))
+		*gDepthBufferCulling = settings.EnableDepthBufferCulling;
+	if (auto _tt = Util::HoverTooltipWrapper()) {
+		ImGui::Text(
+			"Enables a depth buffer culling solution that checks object bounds against the depth buffer before rendering. "
+			"Provides a significant performance boost and includes fixes for game engine bugs. ");
+	}
+
+	if (settings.EnableDepthBufferCulling) {
+		if (ImGui::SliderFloat("Min Occludee Box Extent", &settings.MinOccludeeBoxExtent, 0.1f, 500.0f, "%.1f"))
+			*gMinOccludeeBoxExtent = settings.MinOccludeeBoxExtent;
+		if (auto _tt = Util::HoverTooltipWrapper()) {
+			ImGui::Text(
+				"Sets the minimum bounding box size to use for objects when testing them against the depth buffer. "
+				"Helps prevent small objects from flickering due to precision issues. "
+				"Lower values will give better performance. ");
+		}
+	}
+}
+
+void VR::LoadSettings(json& o_json)
+{
+	settings = o_json;
+}
+
+void VR::SaveSettings(json& o_json)
+{
+	o_json = settings;
+}
+
+void VR::RestoreDefaultSettings()
+{
+	settings = {};
+}
+
+void VR::PostPostLoad()
+{
+	gDepthBufferCulling = reinterpret_cast<bool*>(REL::Offset(0x1EC6B88).address());
```

Comment:
Should probably get these converted to IDs when possible.

Reply:
These global variables are VR only and don't exist in SE/AE so there's no corresponding ID to map to, is there a way to handle this?

Reply:
No need then. I did define in the spec that it'd be the raw integer value of the offset, but it's not necessary.